test/Reachability/unit_solve_hybrid_bouncing_ball.jl

```diff
@@ -31,11 +31,13 @@ HS = HybridSystem(a, m, r, s);
 # initial condition in mode 1
 X0 = Hyperrectangle(low=[10, 0.0], high=[10.2, 0.0]);
 
+X_loc1 = Hyperrectangle(low=[9.8, 0.0], high=[10, 0.0]);
+
 system = InitialValueProblem(HS, X0);
 input_options = Options(:mode=>"reach");
 
-problem_options = Options(:vars=>[1,2], :T=>5.0, :δ=>0.1, :plot_vars=>[1, 2],
-                          :max_jumps=>1, :verbosity=>1);
+problem_options = Options(:vars=>[1,2], :T=>10.0, :δ=>0.1, :plot_vars=>[1, 2],
+                          :max_jumps=>10, :verbosity=>1, :init=>[(1, X_loc1)]);
```

Comment:
Do we need to have that many jumps? I was reducing the time and jumps on purpose to have cheaper tests.